commit cb84e2b92a5d5792f1de8f1a4625819afe28d9bb (refs/changes/32/5232/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-12-14T09:51:49-08:00 (10 months ago)
    
    joyent/node-manta#361 expires-relative short option has wrong case in man page

diff --git a/docs/man/msign.md b/docs/man/msign.md
index 09d3d62..53285a4 100644
--- a/docs/man/msign.md
+++ b/docs/man/msign.md
@@ -1,4 +1,4 @@
-msign 1 "August 2018" Manta "Manta Commands"
+msign 1 "December 2018" Manta "Manta Commands"
 =======================================
 
 NAME
@@ -55,7 +55,7 @@ OPTIONS
   Signed URL should last until EXPIRATION (seconds since epoch).  Default is 1
   hour from `now`.
 
-`-e, --expires-relative expiration`
+`-E, --expires-relative expiration`
   Signed URL should last until EXPIRATION (relative time spec).  Default is 1
   hour from `now`.  Time specification format:
 
diff --git a/man/man1/msign.1 b/man/man1/msign.1
index 674f45d..6b2fca6 100644
--- a/man/man1/msign.1
+++ b/man/man1/msign.1
@@ -1,4 +1,4 @@
-.TH msign 1 "August 2018" Manta "Manta Commands"
+.TH msign 1 "December 2018" Manta "Manta Commands"
 .SH NAME
 .PP
 msign \- create a signed URL to a Manta object
@@ -56,7 +56,7 @@ Authenticate as account (login name).
 Signed URL should last until EXPIRATION (seconds since epoch).  Default is 1
 hour from \fB\fCnow\fR\&.
 .TP
-\fB\fC\-e, \-\-expires\-relative expiration\fR
+\fB\fC\-E, \-\-expires\-relative expiration\fR
 Signed URL should last until EXPIRATION (relative time spec).  Default is 1
 hour from \fB\fCnow\fR\&.  Time specification format:
 .PP
